app/src/main/java/com/android/voyageur/ui/search/DiscoverContent.kt

```diff
@@ -53,7 +56,7 @@ fun DiscoverContent(
   } else {
     HorizontalPager(state = pagerState, modifier = modifier.fillMaxSize().testTag("pager")) { page
       ->
-      TripCard(trip = trips[page], userViewModel = userViewModel)
+      TripCard(trip = trips[page], tripsViewModel, navigationActions, userViewModel = userViewModel)
```

Comment:
nit: for consistency, call TripCard with `tripsViewModel = tripsViewModel, navigationActions = navigationActions` (the same way we pass `trip = trips[page]`

Reply:
I changed this, thanks for the suggestion!